README.md

```diff
@@ -73,18 +73,20 @@ The below property names can be changed in the following ways
+| terraform_binary_path                    | TERRAFORM_BINARY_PATH              | Terraform available on syspath                   | The path to the terraform binary                                                                                      |
+| terraform.root.module.directory          | TERRAFORM_ROOT_MODULE_DIRECTORY    | /tmp on Linux<br/>\AppData\Local\Temp on Windows | The path to the parent directory where all terraform module directories will be stored at as subdirs                  |
+| log.terraform.stdout.stderr              | LOG_TERRAFORM_STDOUT_STDERR        | true                                             | Controls if the command execution output must be logged. If disabled, the output is only returned in the API response |
+| terraform.log.level                      | TERRAFORM_LOG_LEVEL                | INFO                                             | Controls the log level of the terraform binary. Allowed values are INFO, DEBUG, TRACE, WARN and ERROR                 |
+| authorization.token.type                 | AUTHORIZATION_TOKEN_TYPE           | JWT                                              | Authorization server authentication Type, allowed values: OpaqueToken or JWT                                          |
+| authorization.server.endpoint            | AUTHORIZATION_SERVER_ENDPOINT      |                                                  | The endpoint value of the authorization server                                                                        |
+| authorization.api.client.id              | AUTHORIZATION_API_CLIENT_ID        |                                                  | The ID value of the authorization server API client                                                                   |
+| authorization.api.client.secret          | AUTHORIZATION_API_CLIENT_SECRET    |                                                  | The secret value of the authorization server API client                                                               |
+| authorization.swagger.ui.client.id       | AUTHORIZATION_SWAGGER_UI_CLIENT_ID |                                                  | The ID value of the authorization server swagger-ui client                                                            |
+| otel.exporter.otlp.endpoint              | OTEL_EXPORTER_OTLP_ENDPOINT        | http://localhost:4317                            | URL of the OTEL collector                                                                                             |
+| clean.workspace.after.deployment.enabled | OTEL_EXPORTER_OTLP_ENDPOINT        | true                                             | Whether to clean up the workspace after deployment is done,allowed values: true or false. Default value is true       |
```

Comment:
@baixinsui the value in second column must be updated accordingly.

Reply:
Updated.